app/v1/models/response/ItsaStatusDetails.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.models.response
+
+import play.api.libs.functional.syntax._
+import play.api.libs.json.{JsPath, Json, OWrites, Reads}
+import v1.models.domain.StatusReasonEnum
+
+case class ItsaStatusDetails(submittedOn: String, status: String, statusReason: StatusReasonEnum, businessIncomePriorTo2Years: Option[BigDecimal])
+
+object ItsaStatusDetails {
+
+  implicit val writes: OWrites[ItsaStatusDetails] = Json.writes[ItsaStatusDetails]
+
+  implicit val reads: Reads[ItsaStatusDetails] = (
```

Comment:
I don't think this needs the "longhand" reads here, can just be a one-liner, as (AFAICS) see field names and types all match up.